openfeature/provider.go

```diff
@@ -190,3 +196,55 @@ type InterfaceResolutionDetail struct {
 type Metadata struct {
 	Name string
 }
+
+// TrackingEventDetails provides a tracking details with float64 value
+type TrackingEventDetails struct {
+	value  float64
+	fields map[string]interface{}
```

Comment:
The evaluation context uses `attributes` instead of fields. I _think_ maybe we should be consistent? What do you think?

https://github.com/open-feature/go-sdk/blob/13444e73dec4c5a14136e6b88d776c68527d7963/openfeature/evaluation_context.go#L15C2-L15C12

Reply:
That sounds great to me. The main reason why I choose the name `fields` is because that is the word I came up when reading [the specification](https://github.com/open-feature/spec/commit/cd99c3560a68c1810971be19b87ea803fb9b0b3a#diff-1443054d965453435ba3da2907dabbc282d2f808c3a64800f10e05fab8351b1aR1109).

Reply:
I have updated to attributes